README.md

```diff
@@ -0,0 +1,118 @@
+# pds.api-client
+Federated PDS API which provides actionable end points standardized
+between the different nodes.
+
+
+This Python package is automatically generated by the [OpenAPI Generator](https://openapi-generator.tech) project:
+
+- API version: 0.3.dev
+- Package version: 0.6.1
+- Build package: org.openapitools.codegen.languages.PythonLegacyClientCodegen
+
+## Requirements.
+
+Python 2.7 and 3.4+
+
+## Installation & Usage
+### pip install
+
+If the python package is hosted on a repository, you can install directly using:
+
+```sh
+pip install git+https://github.com/GIT_USER_ID/GIT_REPO_ID.git
```

Comment:
You could add the actual url (https://github.com/NASA-PDS/pds-api-client.git) here

Reply:
This file is generated automatically so it should not be in the repository. Let me remove it.